packages/ui/Image/Image.js

```diff
@@ -132,12 +124,22 @@ const ImageBase = kind({
 		 * @default 'fill'
 		 * @public
 		 */
-		sizing: PropTypes.oneOf(['fit', 'fill', 'none'])
+		sizing: PropTypes.oneOf(['fit', 'fill', 'none']),
+
+		/**
+		 * String value or Object of values used to determine which image will appear on
+		 * a specific screenSize.
+		 *
```

Comment:
Should we update the description of the `placeholder` prop to say that it will be used if `src` is not specified?